README.md

```diff
@@ -114,12 +114,18 @@ name                  | description                                | default val
 eclair.datadir        | Path to the data directory                 | ~/.eclair
 eclair.headless       | Run eclair without a GUI                   | 
 eclair.printToConsole | Log to stdout (in addition to eclair.log)  |
+logback.configurationFile | The [`logback`](https://logback.qos.ch) configuration file to use |
 
 For example, to specify a different data directory you would run the following command:
 ```shell
 java -Declair.datadir=/tmp/node1 -jar eclair-node-gui-<version>-<commit_id>.jar
 ```
 
+To use a different logging configuration, and override the internal logback.xml, run:
+```shell
+java -Dlogback.configurationFile=logback-custom.xml -jar eclair-node-gui-<version>-<commit_id>.jar
```

Comment:
It should be an absolute path, like `-Dlogback.configurationFile=/path/to/myconfig.xml`